deployments/ansible/roles/collector/tasks/win_install.yml

```diff
@@ -2,11 +2,10 @@
 
 - name: Install Splunk OpenTelemetry Collector with msi package manager
   ansible.builtin.import_tasks: otel_win_install.yml
-  when: ansible_os_family == "Windows"
```

Comment:
These checks here were redundant since the same test needs pass for `win_install.yml`  to be included.